src/aks-preview/azext_aks_preview/commands.py

```diff
@@ -220,6 +220,8 @@ def load_command_table(self, _):
                               table_transformer=aks_show_nodepool_snapshot_table_format)
         g.custom_command('create', 'aks_nodepool_snapshot_create',
```

Comment:
CRUD nodepool snapshot are not long running operations, so `supports_no_wait=True` is meanlingless.

Reply:
removed